EIPS/eip-5267.md

```diff
@@ -0,0 +1,134 @@
+---
+eip: 5267
+title: Retrieval of EIP-712 domain
+description: A way to describe and retrieve an EIP-712 domain to securely integrate EIP-712 signatures.
+author: Francisco Giordano (@frangio)
+discussions-to: https://ethereum-magicians.org/t/eip-5267-retrieval-of-eip-712-domain/9951
+status: Draft
+type: Standards Track
+category: ERC
+created: 2022-07-14
+requires: 712
```

Comment:
```suggestion
requires: 155, 712, 2612
```
```suggestion
requires: 712
```

Reply:
What is the semantics of the requires field? EIP-2612 is definitely not a requirement, it's just used as an example. EIP-155 is arguably not a requirement either as the chainId doesn't need to be in use.